orsopy/fileio/orso.py

```diff
@@ -182,7 +182,8 @@ def __eq__(self, other: 'OrsoDataset'):
 def save_orso(
         datasets: List[OrsoDataset],
         fname: Union[TextIO, str],
-        comment: Optional[str] = None) -> None:
+        comment: Optional[str] = None,
+        data_separator: str = '') -> None:
```

Comment:
`data_separator` needs to be added to the docstring. Are there any separators which would cause havoc? Should there be a restricted set allowed, `{'\r', '\r\n', '', '\n')`? If there isn't a restricted set then it would be easy to write a non-conformant dataset, e.g. if someone provided `'abc'` then the array creation would fail when it tried to create an array.